util/movementflow.lua

```diff
@@ -0,0 +1,37 @@
+local addonName = ... ---@type string
+
+---@class BetterBags: AceAddon
+local addon = LibStub('AceAddon-3.0'):GetAddon(addonName)
+
+---@class Constants: AceModule
+local const = addon:GetModule('Constants')
+
+---@class movementFlow: AceModule
```

Comment:
Module classes start with upper case. Note, this is for the class name, not the variable.